### DISCUSSION TITLE
why form handleRequest (POST mode) very slow with big form ?

### DISCUSSION THREAD

**Discussion by stloc** [symfony/symfony#50801] [Q&A]
Hello,
with a big form class and his entity (200 fields, no doctrine) on dev symfony server, POST $form->handleRequest($request) take a very long time (15 seconds) , why and how to improve this ?
On GET mode form populate is fast

![image](https://github.com/symfony/symfony/assets/4832070/e4abb4b9-d408-4389-b3fb-04563321166a)

**Comment by MatTheCat** [comment]
Hello,
it seems that all your controller does is sending HTTP requests. What does it have to do with your question?

Anyways nobody will be able to provide an answer unless your share your application code.

**Comment by stloc** [comment]
my controller do somes small actions but after debug it is form handleRequest the problem (POST action only) but this part is not analysed in profiler

```
$form->handleRequest($request); // !!!!
if ($form->isSubmitted() && $form->isValid()) {
```

![image](https://github.com/symfony/symfony/assets/4832070/7024254d-f0fd-432f-82c1-34d04337614b)

**Comment by ghost** [comment]
In the Performance tab change threshold to 0 to see more details

**Comment by stloc** [comment]
no further details

![image](https://github.com/symfony/symfony/assets/4832070/687e1405-7c51-4f0b-83de-38c5fe1da815)

**Reply by stof** [reply]
use a profiling tool able to see _all_ code instead of this performance panel which shows what is explicitly instrumented with the Stopwatch component.

**Reply by stloc** [reply]
thanks for the advice
